### ISSUE TITLE
[CI/CD] Migrate prowjobs out of Neighbor's Prow - Hasselhoff

### ISSUE THREAD

**Issue by KacperMalachowski** [kyma-project/test-infra#9493]
**Description**
The prow jobs owned by the hasselhoff team have to be removed and if necessary substituted with something else.
To simplify the process here is a list of prow jobs that should be migrated by hasselhoff team.

  - [x] pre-busola-backend-deployment-check 
  - [x] pull-busola-integration-cluster-k3d 
  - [x] pull-busola-integration-namespace-k3d 
  - [x] pre-busola-web-deployment-check 
  - [x] pull-busola-lighthouse 
  - [x] pre-kyma-dashboard-dev 
  - [x] pull-kyma-dashboard-integration 
  - [x] pre-kyma-dashboard-prod 
  - [x] pull-kyma-dashboard-smoke-test-prod 
  - [x] pull-kyma-dashboard-smoke-test-stage 
  - [x] pre-kyma-dashboard-stage 
  - [x] post-kyma-dashboard-dev 
  - [x] post-kyma-dashboard-prod 
  - [x] post-kyma-dashboard-stage

**Comment by Sawthis** [comment]
@valentinvieriu what is the status for @kyma-project/hasselhoff team about migration outside of Prow? The deadline was moved from end of January to end of February 2024.

**Comment by Sawthis** [comment]
@mrCherry97 could you provide the update for the issue? The deadline is end of August.

**Comment by mrCherry97** [comment]
Hi @Sawthis 
I have marked jobs that we have moved, here we have our issue for this with all of the details: https://github.com/kyma-project/kyma-dashboard/issues/306 
All test workflows are moved to GithubActions, local build in the busola, and dev build in the kyma-dashboard. We need to adjust the web and backend build in Busola, stage, and prod build in the kyma-dashboard.
We will plan the continuation of this task next week, so we should be fine, and out of prow till the end of August.
